src/ResourceManager/Sql/Commands.Sql/Auditing/Model/AuditingPolicyModel.cs

```diff
@@ -46,11 +46,6 @@ public abstract class AuditingPolicyModel
         public string ServerName { get; set; }
 
         /// <summary>
-        /// Gets or sets the audit type
-        /// </summary>
-        public AuditType AuditType { get; set; }
```

Comment:
@cormacpayne  The problem is that the old cmdlets can return either a table auditing policy, or a blob auditing policy, so it's either ServerAuditingPolicyModel or ServerBlobAuditingPolicyModel for server cmdlets for example. So it cannot be done with the current implementation.